mne/preprocessing/tests/test_ica.py

```diff
@@ -128,6 +128,32 @@ def test_ica_rank_reduction():
         assert_true(n_components < n_pca_components <= rank_after <=
                     rank_before)
 
+@requires_sklearn
+def test_ica_reset():
+    """Test ICA resetting"""
+    raw = io.Raw(raw_fname, preload=True).crop(0, stop, False).crop(0.5)
```

Comment:
Copied from elsewhere, works. So let's keep it as is.

2015-05-13 13:57 GMT+02:00 Alexandre Gramfort <notifications@github.com>:

> In mne/preprocessing/tests/test_ica.py
> <https://github.com/mne-tools/mne-python/pull/2090#discussion_r30223460>:
>
> > @@ -128,6 +128,32 @@ def test_ica_rank_reduction():
> >          assert_true(n_components < n_pca_components <= rank_after <=
> >                      rank_before)
> >
> > +@requires_sklearn
> > +def test_ica_reset():
> > +    """Test ICA resetting"""
> > +    raw = io.Raw(raw_fname, preload=True).crop(0, stop, False).crop(0.5)
>
> why do you need to crop twice?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2090/files#r30223460>.
>